docs/Friendly_User/Applications/amrwind.md

```diff
@@ -0,0 +1,165 @@
+AMR-Wind
+========
+
+## AMR-Wind on Kestrel
+
+AMR-Wind is a massively parallel, block-structured adaptive-mesh,
+incompressible flow solver for wind turbine and wind farm
+simulations. The primary applications for AMR-Wind are: performing
+large-eddy simulations (LES) of atmospheric boundary layer (ABL)
+flows, simulating wind farm turbine-wake interactions using actuator
+disk or actuator line models for turbines, and as a background solver
+when coupled with a near-body solver (e.g., Nalu-Wind) with overset
+methodology to perform blade-resolved simulations of multiple wind
+turbines within a wind farm. For more information see
+https://github.com/Exawind/amr-wind
```

Comment:
Add link text, "[the AMR-Wind documentation](https://github.com/Exawind/amr-wind)" for example